tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,12 +15,15 @@ details: |
     list and deletion.
 
 prepare: |
-    echo "Given openvswitch is installed"
-    apt install -y --no-install-recommends openvswitch-switch
+    # Note that the order of the installs is to avoid a timeout issue on the 
```

Comment:
mmm it's a bit concerning IMO if a deb package installation can timeout depending on a snap being installed or not, given that this test snap has strict confinement and it is installed without `--devmode`. Any ideas of what might be going on?